ecosystem/indexer-grpc/indexer-grpc-utils/src/constants.rs

```diff
@@ -1,5 +1,10 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+/// The maximum number of transactions that can be stored in a blob.
 pub const BLOB_STORAGE_SIZE: usize = 1_000;
+/// GRPC request metadata key for the token ID.
 pub const GRPC_AUTH_TOKEN_HEADER: &str = "x-aptos-data-authorization";
+/// GRPC request metadata key for the request name. This is used to identify the
+/// data destination.
+pub const GRPC_REQUEST_NAME_HEADER: &str = "x-aptos-request-name";
```

Comment:
What's an example of the request name? Does it always need to be specified by the client?

Reply:
if not specified, it's default to `unknown`. An example is `default_processor`